lib/game.rb

```diff
@@ -0,0 +1,144 @@
+require './lib/card'
+require './lib/deck'
+require './lib/turn'
+require './lib/player'
+require 'pry'
+
+class Game
+  attr_accessor :turn,
+                :turn_type,
+                :hand_winner,
+                :full_deck,
+                :deck1,
+                :deck2,
+                :player1,
+                :player2, 
+                :counter
+
+  def initialize
+    @turn = nil
+    # @turn_type = @turn.type
+    # @hand_winner = @turn.winner
+    @full_deck = []
+    @deck1 = nil
+    @deck2 = nil
+    @player1 = nil
+    @player2 = nil
+    @counter = 1
+  end
+
+  def start
+    create_full_deck
+    shuffle_the_deck
+    create_players
+    puts ''
+    puts 'Welcome to War! (or Peace) This game will be played with 52 cards.'
+    puts "The players today are #{player1.name} and #{player2.name}."
+    puts "Press 'RETURN' to start the game"
+    puts '------------------------------------------------------------------'
+    gets
+    puts ''
+    puts 'here we go!'
+    puts ''
+    puts ''
+  end
+
+  def create_full_deck
+    values = %w[2 3 4 5 6 7 8 9 10 Jack Queen King Ace]
+    ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14]
+    values_and_ranks = values.zip(ranks)
+    values_and_ranks.map do |x|
+      full_deck << [x, :diamond].flatten
+      full_deck << [x, :heart].flatten
+      full_deck << [x, :spade].flatten
+      full_deck << [x, :club].flatten
+    end
+
+    @full_deck = full_deck.map do |x|
+      Card.new(x[2], x[0], x[1])
+    end
+  end
+
+  def shuffle_the_deck
+    @full_deck = @full_deck.shuffle
+  end
+
+  def split_deck
+    @deck1 = Deck.new(full_deck.slice!(0, 26))
+    @deck2 = Deck.new(full_deck)
+  end
+
+  def create_players
+    @player1 = Player.new('Megan', @deck1)
+    @player2 = Player.new('Aurora', @deck2)
+  end
+
+  def game_over?
+    @turn.player1.has_lost? || @turn.player2.has_lost?
```

Comment:
```suggestion
    @player1.has_lost? || @player2.has_lost?
```
I'm pretty sure you can check the player instance variable directly without relying on the turn. The players are going to be the same no matter what. Every game will have lots of turns so lets use the thing that is always there.